openmetadata-docs/content/v1.1.0/deployment/docker/index.md

```diff
@@ -289,26 +298,20 @@ OPENMETADATA_HEAP_OPTS="-Xmx2G -Xms2G"
 
 The flag `Xmx` specifies the maximum memory allocation pool for a Java virtual machine (JVM), while `Xms` specifies the initial memory allocation pool.
 
-Restart the OpenMetadata Docker Compose Application using `docker compose --env-file my-env-file -f docker-compose.yml up -d` which will recreate the containers with new environment variable values you have provided.
-
-# Production Deployment
-
-If you are planning on going to PROD, we also recommend taking a look at the following
-other deployment  strategies:
-
-{%inlineCalloutContainer%}
-  {%inlineCallout
-    color="violet-70"
-    icon="storage"
-    bold="Deploy on Bare Metal"
-    href="/deployment/bare-metal"%}
-    Deploy OpenMetadata directly using the binaries.
-  {%/inlineCallout%}
-  {%inlineCallout
-    color="violet-70"
-    icon="fit_screen"
-    bold="Deploy on Kubernetes"
-    href="/deployment/kubernetes"%}
-    Deploy and scale with Kubernetes
-  {%/inlineCallout%}
-{%/inlineCalloutContainer%}
+Restart the OpenMetadata Docker Compose Application using `docker compose --env-file <my-env-file> -f docker-compose.yml up --detach` which will recreate the containers with new environment variable values you have provided.
+
+## Security
+
+Please follow our [Enable Security Guide](/deployment/docker/security) to configure security for your OpenMetadata
+installation.
+
+## Advanced
+
+If you want to persist your data, prepare [Named Volumes](/deployment/docker/volumes) for the containers.
```

Comment:
This should be default no? why are we putting this in advanced section

Reply:
It's default for MySQL, PostgreSQL, Ingestion, ElasticSearch Containers we ship! I'll move this section to specifically point to adding Docker Volumes to OpenMetadata Server.

Reply:
I've refactored the docs to remove the noisy and unnecessary volumes section! Instead Added a small advanced section which takes an example to setting jwt configurations and adding files as volumes to openmetadata server!